src/main/java/seedu/address/logic/Logic.java

```diff
@@ -46,6 +46,8 @@ public interface Logic {
      */
     ObservableList<Person> getFilteredPersonList();
 
+    ObservableList<Person> getSortedPersonList();
```

Comment:
A bit overkill to return sortedPersonList just to find out how many contacts you have, probably should have one method that just returns that instead (Tell, don't ask). But since we rushing for time we close one eye.